dists/fs-repo-migrations/matrices/v1.0.0

```diff
@@ -0,0 +1,10 @@
+darwin 386
```

Comment:
(to be clear, i understand that osx has been supposed to run only on `amd64` for some time now, but have seen this specific problem. Until golang disallows `darwin 386`, I think we should continue to have it.